### ISSUE TITLE
Error: The system cannot find the path specified: 'C:\\Users\\ram\\AppData\\Local\\Temp\\resources/text'

### ISSUE THREAD

**Issue by thomsonian2023** [codelucas/newspaper#828]
I am using the sample code:

`from newspaper import Article
url = 'http://fox13now.com/2013/12/30/new-year-new-laws-obamacare-pot-guns-and-drones/'
article = Article(url)
article.download()
article.html
article.parse()
article.authors
article.publish_date`

I get the following error:
`File "C:\Program Files\Python38\lib\site-packages\newspaper\utils.py", line 346, in get_available_languages
    stopword_files = os.listdir(os.path.join(settings.STOPWORDS_DIR))
FileNotFoundError: [WinError 3] The system cannot find the path specified: 'C:\\Users\\ram\\AppData\\Local\\Temp\\resources/text'`

**Comment by Constuelo** [comment]
It's pretty obvious when you see it `C:\\Users\\ram\\AppData\\Local\\Temp\\resources/text`

Have another look at that path

**Comment by thomsonian2023** [comment]
I don't understand your answer. What part of source do I modify so that it can work around that error?

**Comment by KilledMufasa** [comment]
The path you specified is: 

`C:\\Users\\ram\\AppData\\Local\\Temp\\resources/text`

The correct one is probably:
`C:/Users/ram/AppData/Local/Temp/resources/text`

And perhaps you're missing a file extension (e.g. .txt)? 

You can paste the link in a browser or in file explorer to see if the path is correct
